src/app/apiAndObjects/objects/enums/colourGradientType.enum.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line no-shadow
+export enum ColourGradientType {
```

Comment:
This enum has really local scope, specifically to the color ramps on those maps, so it would make sense to keep the enum in a more local context, potentially with the actual ramp color codes.

Reply:
It's used in the matSettingsDialog.component.ts, and dialogService too?

Reply:
Ah, I see.  Could still be moved into the map component folder as it only gets passed through the dialog code, not used.